client/src/components/Modal.jsx

```diff
@@ -95,24 +105,22 @@ class Modal extends React.Component {
   render() {
     const leftButton = '<';
     const rightButton = '>';
-    const { pictures, currentPicture } = this.state;
+    const { pictures, currentPosition, transform } = this.state;
     return (
       <div className="modal" tabIndex="0" onKeyDown={e => this.onModalKeyPress(e)}>
         <div className="x-button" onClick={e => this.onButtonClick(e)}>x</div>
         <div className="right-button" onClick={e => this.onRightClick(e)}>{rightButton}</div>
         <div onClick={e => this.onLeftClick(e)} className="left-button">{leftButton}</div>
-        <div className='main-container'>
-          <div className='container-center'>
-            <ModalPhoto url={this.state.currentUrl} />
-            <Description isVerified={this.state.currentPicture.isVerified} position={this.state.currentPosition + 1} size={this.state.length0 + 1} description={this.state.currentPicture.description} />
-            <div className={`photo-carousel active-${this.state.currentPosition}`} style={{ opacity: 1 }}>
```

Comment:
if you want to leave this alone to move on to the proxy server stuff that's fine, but i think that the style stuff should really go with the rest of the css you have if possible. was there a reason you kept this styling separate?